recipes/linuxbridge.rb

```diff
@@ -22,7 +22,7 @@
 node_type = node['osl-openstack']['node_type']
 int_mappings = []
 node['osl-openstack']['physical_interface_mappings'].each do |int|
-  interface = if int[node_type][node['fqdn']]
+  interface = if int[node_type][node['fqdn']] # rubocop:disable Style/RedundantCondition
                 int[node_type][node['fqdn']]
               else
                 int[node_type]['default']
```

Comment:
Would this be better served by :arrow_down: instead of this conditional? That way we don't have to disable the cop.
```rb
interface = int[node_type][node['fqdn']] || int[node_type]['default']
```
(same for `vxlan_interface`)